sql/create_shards.sql.tmpl

```diff
@@ -13,12 +13,38 @@ CREATE FUNCTION create_table_then_fail(cstring, integer)
 	LANGUAGE C STRICT;
 
 -- ===================================================================
+-- create test type, operator, operator family and operator class
+-- ===================================================================
+CREATE TYPE dummy_type AS (
+    i integer
+);
+
+CREATE FUNCTION dummy_type_function(dummy_type, dummy_type) RETURNS boolean
+AS ' SELECT TRUE;'
+LANGUAGE SQL
+IMMUTABLE
+RETURNS NULL ON NULL INPUT;
+
+CREATE OPERATOR = (
+    LEFTARG = dummy_type,
+    RIGHTARG = dummy_type,
+    PROCEDURE = dummy_type_function);
```

Comment:
Can you put the closing parenthesis on the next line (as you did with `CREATE TYPE`)?